### PR TITLE
Quick reply button UI

### PR DESCRIPTION
**Pull request by EgorovEI**
Added QuickReplyView which displays left-aligned button's grid
Added handling QR button actions
Added displaying QR messages
Added style for QR button

MOB-2395

### CODE REVIEW

**Review by rasmustautsglia — APPROVED** [review]

**Comment by EgorovEI** [comment]
!squash

File: GliaWidgets/Sources/View/Chat/ChatStyle.swift [hunk 1]
```diff
@@ -49,6 +49,9 @@ public class ChatStyle: EngagementStyle {
 
     public var systemMessage: SystemMessageStyle
 
+    /// Style for Quick Reply buttons.
+    public var quickReplyButtonStyle: GvaQuickReplyButtonStyle
```
[the remark this answers is not in the document]
Comment (EgorovEI) [reply GliaWidgets/Sources/View/Chat/ChatStyle.swift]: Done

File: GliaWidgets/Sources/View/Chat/GVA/QuickReply/QuickReplyView.swift [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        collectionView.dataSource = self
+
+        addSubview(collectionView)
+        collectionView.translatesAutoresizingMaskIntoConstraints = false
+
+        let height = collectionView.heightAnchor.constraint(equalToConstant: 0)
+        height.priority = .required
+        collectionViewHeightConstraint = height
+
+        NSLayoutConstraint.activate([
```
Comment (yurii-glia) [inline GliaWidgets/Sources/View/Chat/GVA/QuickReply/QuickReplyView.swift:63]: if you will inherit from BaseView, you can establish constraints in defineLayout function.

**Review by yurii-glia — APPROVED** [review]

File: GliaWidgets/Sources/View/Chat/GVA/QuickReply/QuickReplyView.swift [hunk 3]
```diff
@@ -0,0 +1,114 @@
+import UIKit
+
+final class QuickReplyView: UIView {
+
```
[the remark this answers is not in the document]
Comment (EgorovEI) [reply GliaWidgets/Sources/View/Chat/GVA/QuickReply/QuickReplyView.swift]: Fixed 0270b3ef7b17e734cbe3424c77ed8fe16513f45f

File: GliaWidgets/Sources/View/Chat/GVA/QuickReply/QuickReplyView.swift [hunk 4]
```diff
@@ -0,0 +1,114 @@
+import UIKit
+
+final class QuickReplyView: UIView {
```
[the remark this answers is not in the document]
Comment (EgorovEI) [reply GliaWidgets/Sources/View/Chat/GVA/QuickReply/QuickReplyView.swift]: Fixed 0270b3ef7b17e734cbe3424c77ed8fe16513f45f

File: GliaWidgets/Sources/View/Chat/GVA/QuickReply/QuickReplyView.swift (hunk 2 above)
Comment (EgorovEI) [reply GliaWidgets/Sources/View/Chat/GVA/QuickReply/QuickReplyView.swift:52]: Fixed 0270b3ef7b17e734cbe3424c77ed8fe16513f45f

**Comment by EgorovEI** [comment]
!squash

**Pull request merged**